src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -160,6 +162,8 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
             builder.AppendLine($"#bantime=<number>");
             builder.AppendLine($"#The maximum number of outbound connections. Default {ConnectionManagerSettings.DefaultMaxOutboundConnections}.");
             builder.AppendLine($"#maxoutboundconnections=<number>");
+            builder.AppendLine($"#The maximum number of inbound connections. Default {ConnectionManagerSettings.DefaultMaxConnections}.");
```

Comment:
This should be "The maximum number of connections to peers. Default..." from StratisX.